core/client/fs/src/main/java/alluxio/client/file/options/ExistsOptions.java

```diff
@@ -34,21 +37,46 @@ public static ExistsOptions defaults() {
   }
 
   private ExistsOptions() {
-    // No options currently
+    mLoadMetadataType = LoadMetadataType.Once;
```

Comment:
should this use `Configuration.getEnum(PropertyKey.USER_FILE_METADATA_LOAD_TYPE, LoadMetadataType.class)` as well?